libs/designer/src/lib/core/state/panelV2/panelSlice.ts

```diff
@@ -97,24 +98,53 @@ export const panelSlice = createSlice({
       state.discoveryContent.selectedOperationGroupId = '';
       state.discoveryContent.isAddingTrigger = false;
     },
-    clearPanel: (state) => {
+    clearPanel: (state, action: PayloadAction<{ clearPinnedState?: boolean }>) => {
```

Comment:
Are we able to make this action have an optional payload?
So we don't have to call `clearPanel({})` with an empty object

Reply:
I'm not sure if I can since I think `PayloadAction` has to be provided if specified even if I use something like `| undefined`, but let me see what I can do.